include/Core/BulkDelete.php

```diff
@@ -474,6 +475,9 @@ private function get_delete_users_admin_page() {
 
 		$users_page->add_module( new DeleteUsersByUserRoleModule() );
 		$users_page->add_module( new DeleteUsersByUserMetaModule() );
+		if ( class_exists( 'BuddyPress' ) ) {
```

Comment:
Add an additional check to make sure that this is a single site install and not multisite.

Reply:
Fixed.